magento/magento2
# Change product_price_value in cart data section based on tax settings

### Description (*)

Change the `product_price_value` data based on the tax setting. Uses the same functions as used in the template for the `product_price` data so it should always match the data shown in the minicart. The exception is when both prices should be shown (inclusive and exclusive tax), in this case a object is returned with both prices in it. `product_price_value` is not used anywhere in Magento so this shouldn't be a problem.

### Fixed Issues (if relevant)

1. magento/magento2#20310: Cart section data has wrong product_price_value

### Manual testing scenarios (*)

1. Set Magento configuration Stores > Configuration > Sales > Tax > Shopping Cart Display Settings > Display Prices to Including Tax/Excluding Tax/Both
2. Add a item to your cart
3. Check the cart data section in dev tools to see if the correct price is shown.

### Contribution checklist (*)
